g2p_registry_membership/models/individual.py

```diff
@@ -16,9 +16,13 @@ class G2PMembershipIndividual(models.Model):
     def _recompute_parent_groups(self, records):
         field = self.env["res.partner"]._fields["force_recompute_canary"]
         for line in records:
-            if line.is_registrant and not line.is_group:
-                groups = line.individual_membership_ids.mapped("group")
-                self.env.add_to_compute(field, groups)
+            if line.is_registrant and not line.is_group and not line.disabled:
+                try:
+                    groups = line.individual_membership_ids.mapped("group")
+                except Exception as e:
+                    _logger.info("_recompute_parent_groups: exception: %s" % e)
+                else:
+                    self.env.add_to_compute(field, groups)
```

Comment:
Isn't this going to create an issue when you disable the line, it will not force the recompute? 
So if there is 4 members, it will not be updated to 3?

Reply:
Please check again @jeremi , I applied fixes to this.